docs/upcoming_changes/9780.change.rst

```diff
@@ -0,0 +1,11 @@
+Disabling `sys.monitoring` support by default
+---------------------------------------------
+
+The `sys.monitoring` support is disabled by default due to compatibility issues 
```

Comment:
```suggestion
The ``sys.monitoring`` support is disabled by default due to compatibility issues 
```
I think these need to be double back ticks throughout as this renders as RST and the subject of the back ticks is to be typeset as code format?